### ISSUE TITLE
Remove application menu to follow the new GNOME HIGs

### ISSUE THREAD

**Issue by nonoesimposible** [gnunn1/tilix#1514]
Hello,

Given the fact that following the GNOME Human Interface Guidelines seems to be an ongoing goal of the project, it would probably be a good idea to move the items from this menu into the hamburger menu as described in the latest guidelines.

The app menu will be removed in GNOME 3.32.

https://wiki.gnome.org/Design/Whiteboards/AppMenuMigration

**Comment by egmontkob** [comment]
(a gnome-terminal developer chiming in)

Do you know what's the current state of this initiative in GNOME?

The page you linked was last modified on 2018-06-26, and says "3.30 ships with app menu support in the shell, but most apps don't use it anymore", whereas a month later https://mail.gnome.org/archives/desktop-devel-list/2018-July/msg00029.html says "No app menu changes for GNOME 3.30, please!"

So I'm wondering, it seems to me that GNOME is already one cycle behind compared to the current proposal, am I right?

**Comment by gnunn1** [comment]
When I asked about it on IRC I was told the plan was deferred to 3.32 and not to make the change for 3.30. Once 3.30 lands in Arch stable I plan on having a look at the status of this across the various gnome apps and make an appropriate decision.

**Comment by gnunn1** [comment]
I've added the app menu items in a first iteration of this, see screenshot below (ignore GC item, it's only active in development).

![screenshot from 2018-09-23 18-21-47](https://user-images.githubusercontent.com/9358238/45933711-43334800-bf60-11e8-9986-b64baa70b5f2.png)

**Comment by gnunn1** [comment]
I think that first iteration meets the requirements, closing this issue as completed.
